src/Microsoft.AspNet.Mvc.Core/Resources.resx

```diff
@@ -445,4 +445,7 @@
   <data name="ApiExplorer_UnsupportedAction" xml:space="preserve">
     <value>The action '{0}' has ApiExplorer enabled, but is using conventional routing. Only actions which use attribute routing support ApiExplorer.</value>
   </data>
+  <data name="CacheProfileNotFound" xml:space="preserve">
+    <value>The '{0}' cache profile is not defined. Please define it in MvcOptions.</value>
```

Comment:
`Please` and `Thank You` are good to learn in kindergarten but we shouldn't be using them in docs :laughing: